### ISSUE TITLE
-rpath now does not work on Darwin systems (at least with Gnu compilers).

### ISSUE THREAD

**Issue by BarrySmith** [DrTimothyAldenDavis/SuiteSparse#81]
Tim,

    We've had multiple reports from PETSc users regarding this and I can reproduce it on my Mac.

in SuiteSparse_config.mk

```
ifeq ($(UNAME), Darwin)
        # To compile on the Mac, you must install Xcode.  Then do this at the
        # command line in the Terminal, before doing 'make':
        # xcode-select --install
        CF += -fno-common
        BLAS ?= -framework Accelerate
        LAPACK ?= -framework Accelerate
        # OpenMP is not yet supported by default in clang
        CFOPENMP =
        LDLIBS += -rpath $(INSTALL_LIB)
    endif
```

```
/usr/local/bin/gmake install INSTALL=/Users/barrysmith/Src/petsc/arch-darwin-c-debug/externalpackages/git.suitesparse
gmake[1]: Entering directory '/Users/barrysmith/Src/petsc/arch-darwin-c-debug/externalpackages/git.suitesparse/SuiteSparse_config'
/Users/barrysmith/Src/petsc/arch-darwin-c-debug/bin/mpicc -dynamiclib -single_module -undefined dynamic_lookup -multiply_defined suppress  -Wall -Wwrite-strings -Wno-strict-aliasing -Wno-unknown-pragmas -fvisibility=hidden -g3  -L/Users/barrysmith/Src/petsc/arch-darwin-c-debug/lib -dynamiclib -compatibility_version 5 -current_version 5.8.1 -Wl,-install_name -Wl,@rpath/libsuitesparseconfig.5.dylib -shared -undefined dynamic_lookup SuiteSparse_config.o -o /Users/barrysmith/Src/petsc/arch-darwin-c-debug/lib/libsuitesparseconfig.5.8.1.dylib -lm -rpath /Users/barrysmith/Src/petsc/arch-darwin-c-debug/lib
gmake[1]: Leaving directory '/Users/barrysmith/Src/petsc/arch-darwin-c-debug/externalpackages/git.suitesparse/SuiteSparse_config'gcc-10: error: unrecognized command-line option '-rpath'
gmake[1]: *** [Makefile:53: /Users/barrysmith/Src/petsc/arch-darwin-c-debug/lib/libsuitesparseconfig.5.8.1.dylib] Error 1
gmake: *** [Makefile:23: library] Error 2
```

**Comment by knepley** [comment]
@BarrySmith Can you try replacing it with
`LDLIBS += -Wl,-rpath,$(INSTALL_LIB}`
It looks like that is what we have in PETSc.

**Comment by DrTimothyAldenDavis** [comment]
This should be fixed in the draft v5.11.0.
